src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -56,6 +55,11 @@ internal static string Resolve(string key)
 
                 if (string.IsNullOrEmpty(value))
                 {
+                    value = ConfigurationManager.AppSettings[key];
+                }
+
+                if (string.IsNullOrEmpty(value))
+                {
                     value = Environment.GetEnvironmentVariable(key);
                 }
             }
```

Comment:
What's supposed to happen if the key doesn't exist here?  Is that still a success case?

---
In reply to: [57023038](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/57#discussion_r57023038) [](ancestors = 57023038)